package.json

```diff
@@ -29,24 +29,16 @@
   "description": "ProcessHub SDK for JavaScript/TypeScript",
   "dependencies": {
     "@hapi/hapi": "^18.4.0",
-    "@hapi/nes": "^11.2.2",
     "bpmn-moddle": "^4.0.0",
-    "chai": "^4.2.0",
-    "connected-react-router": "^6.6.1",
     "core-js": "^3.4.8",
     "fetch-everywhere": "^1.0.5",
-    "history": "^4.10.1",
-    "immutability-helper": "^2.9.1",
     "lodash": "^4.17.15",
     "moment": "^2.24.0",
     "react": "^16.9.0",
-    "react-dom": "^16.9.0",
-    "react-redux": "^7.1.3",
-    "react-router": "^5.1.2",
-    "redux": "^4.0.5",
-    "redux-thunk": "^2.3.0"
+    "react-dom": "^16.9.0"
   },
   "scripts": {
+    "tsc": "tsc",
```

Comment:
Unnecessary, this is simply callable via `npx tsc` now without this entry.

But if you really want it we can keep it.